packages/ibm-products/src/components/CreateTearsheet/CreateTearsheet.tsx

```diff
@@ -152,7 +152,7 @@ export interface CreateTearsheetProps extends PropsWithChildren {
   /**
    * The main title of the tearsheet, displayed in the header area.
    */
-  title?: ReactNode;
+  title?: string;
```

Comment:
This should remain `ReactNode` from what I can tell. Sometimes people have more complex titles that require passing in a node rather than a string.